janisdk/fromwdl/__init__.py

```diff
@@ -0,0 +1,37 @@
+def add_fromwdl_args(parser):
+    parser.description = (
+        "Parse a WDL command line tool / workflow and write it as a Janis file"
+    )
+
+    parser.add_argument(
+        "-o",
+        "--output",
+        help="Directory to output the workflow / tools to, otherwise this is written to stdout",
+    )
+
+    parser.add_argument("wdlfile", help="The path to the WDL file")
+
+    parser.add_argument(
+        "translation", default="janis", choices=["wdl", "wdl", "janis"], nargs="?"
+    )
+
+    return parser
+
+
+def do_fromwdl(args):
+    from janis_core import WdlParser, Logger
+
+
+    Logger.info(f"Loading WDL file: {args.wdlfile}")
+    tool = WdlParser.from_doc(args.wdlfile)
+
+    Logger.info(f"Loaded {tool.type()}: {tool.versioned_id()}")
+
+    translated = tool.translate(
+        args.translation,
+        to_console=args.output is None,
+        to_disk=args.output is not None,
+        export_path=args.output,
+    )
```

Comment:
Hmm.. I was thinking that the "try/except ValidationError" logic would go here. But I see that this function returns the translation, so it might be intended to be used outside of the CLI where `print`ing to stderr would be appropriate. Maybe amend the `ValidationError` class to always include the `pos` and `node` information would be a nicer solution? That way everyone wins.